src/styles/main.scss

```
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import "./utils/normilize";
+@import "./utils/variables";
+@import "./utils/mixins";
```

```suggestion
@import "./utils/variables";
@import "./utils/mixins";
@import "./utils/normilize";
```

Подключай в порядке важности.